modules/tinymce/src/plugins/table/test/ts/module/test/TableTestUtils.ts

```diff
@@ -386,27 +386,33 @@ const sAssertDialogValues = (data, hasAdvanced, generalSelectors) => {
   return sAssertTabContents(data, generalSelectors);
 };
 
-const sAssertTableStructureWithSizes = (editor: Editor, cols: number, rows: number, unit: string | null, tableWidth: number | null, widths: Array<number | null>[],
-                                        options: { headerRows: number; headerCols: number } = { headerRows: 0, headerCols: 0 }) => GeneralSteps.sequence([
-  sAssertTableStructure(editor, ApproxStructure.build((s, str) => s.element('table', {
-    attrs: { border: str.is('1') },
-    styles: { 'border-collapse': str.is('collapse') },
-    children: [
-      s.element('tbody', {
-        children: Arr.range(rows, (rowIndex) => s.element('tr', {
-          children: Arr.range(cols, (colIndex) => s.element(colIndex < options.headerCols || rowIndex < options.headerRows ? 'th' : 'td', {
-            children: [
-              s.either([
-                s.element('br', { }),
-                s.text(str.contains('Cell'))
-              ])
-            ]
-          }))
-        }))
-      })
-    ]
-  }))),
-  Step.sync(() => {
+interface Options {
+  headerRows: number;
+  headerCols: number;
+}
+
+const sAssertTableStructureWithSizes = (
+  editor: Editor,
+  cols: number,
+  rows: number,
+  unit: string | null,
+  tableWidth: number | null,
+  widths: Array<number | null>[],
+  useColGroups: boolean,
+  options: Options = { headerRows: 0, headerCols: 0 }
```

Comment:
No, options have a default setting, while this is set so you need to be explicit whether you use it or not. Adding it to options would either force you to make the headerRows/headerCols, or make useColGroups have a default, something I wanted to avoid.